tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -109,7 +110,9 @@ private TChannel(@NotNull Builder builder) {
         this.timer = builder.timer;
         this.clientMaxPendingRequests = builder.clientMaxPendingRequests;
         this.tracer = builder.tracer;
-        this.tracingContext = builder.tracingContext;
+        this.tracingContext = builder.tracingContext == null
```

Comment:
`tracingContext` now defaults to `OpenTracingContext` backed by native tracer `scopeManager` (instead of useless no-op implementation).